moneriote/rpc.py

```diff
@@ -94,6 +94,8 @@ def cache_read(path):
         nodes = RpcNodeList()
         for node in blob:
             dt = dateutil_parse(node.pop('dt'))
+            if (datetime.now() - dt).total_seconds() > CONFIG['scan_interval']:
+                break
             if (datetime.now() - dt).total_seconds() < CONFIG['scan_interval'] and 'address' in node:
```

Comment:
For days of testing, there are always 3 - 5 nodes stay in nodes cache. It resulted mass-scan was never triggered in days.